daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc/Test.hs

```diff
@@ -102,7 +102,7 @@ prettyErr lfVersion err = case err of
         DA.Pretty.string (show berr)
     SSC.ScenarioError serr ->
         SS.prettyBriefScenarioError
-          (LF.emptyWorld lfVersion)
+          (LF.initWorld [] lfVersion)
```

Comment:
Why inline this here?

Reply:
`emptyWorld` wasn't used anywhere else, and it has to be equivalent to `initWorld []`, so I figured reduction in API surface made up for it. A judgement call. Happy to go back if you want.

Reply:
Makes sense too. Thanks for explaining.